client/layout/sidebar/expandable-heading.jsx

```diff
@@ -4,12 +4,9 @@ import TranslatableString from 'calypso/components/translatable/proptype';
 import SidebarHeading from 'calypso/layout/sidebar/heading';
 import { decodeEntities } from 'calypso/lib/formatting';
 
-const noop = () => {};
-
 const ExpandableSidebarHeading = ( {
 	title,
 	count,
-	onClick,
```

Comment:
Here I'm completely removing the `onClick` prop and I'm letting the component pass it down as part of the `...props` object.

Defaulting to `noop` is not needed because:
1. The only usage of this component (in `ExpandableSidebarMenu`) is always passing `onClick`
2. The inner `SidebarHeading` component can deal with missing `onClick` value, and will behave more correctly when the value is missing: it won't install a noop `onKeyDown` handler, and won't set `tabIndex` to `0`. The component will be inert.